ppr-ui/src/store/store.ts

```diff
@@ -463,12 +467,33 @@ export const useStore = defineStore('assetsStore', () => {
         component: MhrReviewConfirm
       }]
   })
+  const getUserAccessSteps = computed(() => {
```

Comment:
Do we need `computed` here?

Reply:
We do, due to the validation/computed flags that are contained within the steps configurations.
Haven't implemented the flags yet, but we will as filing moves forward. 

We could explore them as resources but i'm not sure how reactice the validation flags would be for our stepper filings.